crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -2126,6 +2144,17 @@ type ServiceConfig {
 	maxMoveValueDepth: Int!
 }
 
+"""
+A shared object is an object that is shared using the 0x2::transfer::share_object function
+and is accessible to everyone.
+Unlike owned objects, once an object is shared, it stays mutable and can be accessed by anyone,
+unless it is made immutable. An example of immutable shared objects are all published packages
```

Comment:
Note that shared objects can never be made immutable -- once an object is shared, it is shared forever, (until it is deleted, with the introduction of shared object deletion), and only a freshly created object can be shared -- I can't remember exactly but I suspect that we may have a similar rule for immutable objects (that you can only freeze a freshly created object, not one that has been owned before).

Reply:
That's good to know, thanks! I sent another PR to make sure this gets corrected.